### DISCUSSION TITLE
Q-select: How to make the menu dense?

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#11333] [General - Components / Directives / etc] [answered]
When q-select is set to 'dense', the drop-down menu is still not dense. Is there any way to make the menu dense too, so it looks like q-list?
![denseList](https://user-images.githubusercontent.com/45148066/141686472-cb1a5255-a9a9-41f6-af61-dda4e003f16a.png)

**Comment by pdanpdan** [comment, marked as answer]
Options-dense
Please use the search in the API card for details

**Reply by CrawfordW** [reply]
Thanks. It works, but I don't see it in the docs API.

**Reply by MilosPaunovic** [reply]
[https://quasar.dev/vue-components/select#qselect-api](https://quasar.dev/vue-components/select#qselect-api)

![Screenshot 2021-11-14 at 16 36 57](https://user-images.githubusercontent.com/62475782/141687805-13805803-4aea-4633-9914-d7b6de9c2d4d.png)

**Reply by CrawfordW** [reply]
OK thanks. Don't know why I missed this. I appreciate your help.
